### ISSUE TITLE
Custom Pitch is Putting Penalty Box Arc in the Wrong location 

### ISSUE THREAD

**Issue by gregorydsam** [Torvaney/ggsoccer#31]
Using a custom pitch with 105x68 dimensions and a centre dot at (0,0)/ origin at (-52.5, 34) is plotting one of the penalty box arcs in the wrong location. This is true regardless of whether `coord_flip` is applied or not. 

```r
pitch_trac_cust <- list(
  length = 105,
  width = 68,
  penalty_box_length = 16.5,
  penalty_box_width = 40.3,
  six_yard_box_length = 5.5,
  six_yard_box_width = 18.32,
  penalty_spot_distance = 11,
  goal_width = 7.32,
  origin_x = -52.5,
  origin_y = -34
)
```

<img width="608" alt="Screen Shot 2023-03-20 at 3 38 35 PM" src="https://user-images.githubusercontent.com/15912975/226448351-4de98cc6-5671-4f61-b49b-d8836b7fdd9a.png">

**Comment by Torvaney** [comment]
Thanks for reporting this, Sam. This has been closed by @Flosch1006 in #34

**Comment by Torvaney** [comment]
``` r
library(ggplot2)
library(ggsoccer)

pitch_trac_cust <- list(
    length = 105,
    width = 68,
    penalty_box_length = 16.5,
    penalty_box_width = 40.3,
    six_yard_box_length = 5.5,
    six_yard_box_width = 18.32,
    penalty_spot_distance = 11,
    goal_width = 7.32,
    origin_x = -52.5,
    origin_y = -34
)

ggplot() +
    annotate_pitch() +
    theme_pitch() +
    coord_flip() +
    scale_y_reverse()
#> Scale for y is already present.
#> Adding another scale for y, which will replace the existing scale.
```

![](https://i.imgur.com/tC2u0Fp.png)

<sup>Created on 2023-04-26 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
